server/platform/services/telemetry/telemetry.go

```diff
@@ -218,6 +238,38 @@ func (ts *TelemetryService) SendTelemetry(event string, properties map[string]an
 	}
 }
 
+func (ts *TelemetryService) SendTelemetryForFeature(featureName TrackFeature, event string, properties map[string]any) {
+	if ts.rudderClient != nil {
+		skus, ok := featureSKUS[featureName]
+		if !ok {
+			skus = []TrackSKU{}
+			mlog.Warn("Telemetry SKUS are not defined for the feature", mlog.String("feature", featureName))
+		}
+		feature := EventFeature{
+			Name: featureName,
+			SKUS: skus,
+		}
+
+		var context *rudder.Context = &rudder.Context{
+			Extra: map[string]any{"feature": feature},
+		}
+		// if we are part of a cloud installation, add it's ID to the tracked event's context
```

Comment:
Typo
```suggestion
		// if we are part of a cloud installation, add its ID to the tracked event's context
```

Reply:
I'll fix that there and in the original where it was copied, thanks for spotting it :)